.github/workflows/e2e.yml

```diff
@@ -52,7 +52,58 @@ jobs:
       - name: Build Uppy packages
         run: corepack yarn build
       - name: Run end-to-end browser tests
-        run: corepack yarn run e2e:ci
+        run: >
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci &&
+          corepack yarn run e2e:ci
```

Comment:
```suggestion
          for run in {1..50}; do corepack yarn run e2e:ci; done
```

Reply:
This is temporary, so it doesn't really matter